pkg/golinters/govet.go

```diff
@@ -41,18 +49,26 @@ func getAllAnalyzers() []*analysis.Analyzer {
 		atomic.Analyzer,
 		atomicalign.Analyzer,
 		bools.Analyzer,
+		buildssa.Analyzer,
```

Comment:
but this analyzer in internal and exists just for building SSA: it reports no issues

Reply:
Can we automatically filter internal analyzers by some condition?

Reply:
I don't know how to do it

Reply:
Seems like the following should work:
```go
func isInternal(a *analysis.Analyzer) bool {
	return a.ResultType != nil
}
```

Reply:
I've created the #763 to address this issue. 
I still want to import every analyzer and explicitly external ones, seems like it is most maintainable approach.